packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -597,29 +591,13 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends React.Componen
    * without shifting focus.
    */
   protected onClick = (ev: React.MouseEvent<HTMLInputElement>): void => {
-    const input = this.input.current ? this.input.current.value : '';
-
     if (this.props.inputProps !== undefined && this.props.inputProps.onClick !== undefined) {
       this.props.inputProps.onClick(ev);
     }
```

Comment:
```suggestion
    this.props.inputProps?.onClick?(ev);
```
I think that would work :)